WordPress/src/main/java/org/wordpress/android/ui/reader/viewmodels/ReaderReadingPreferencesViewModel.kt

```diff
@@ -65,15 +71,24 @@ class ReaderReadingPreferencesViewModel @Inject constructor(
         readingPreferencesTracker.trackItemTapped(fontSize)
     }
 
-    fun saveReadingPreferencesAndClose() {
+    /**
+     * An exit action has been triggered by the user. This means that we need to save the current preferences and emit
+     * the close event, so the dialog is dismissed.
+     */
+    fun onExitActionClick() {
```

Comment:
Ditto here. Just to save some curly braces.

Reply:
Same reasoning for not returning `Job` here.